corehq/form_processor/models/mixin.py

```diff
@@ -0,0 +1,29 @@
+
+
+class IsImageMixin:
```

Comment:
nit (and also out of scope of the PR but just wanted to share thought): this might be better named as `AttachmentMixin`.
Moreover, I think it would be better to just exist as a util function instead. Single function Mixin inherited by 2 classes, seems overkill and a poor use of inheritance/mixin?

Reply:
I agree, but yes, it was out of scope for this PR.